frontend/src/store/event.js

```diff
@@ -0,0 +1,92 @@
+import axios from 'axios'
+import io from 'socket.io-client'
+
+const socket = io(process.env.VUE_APP_SOCKET_PATH)
+
+const mutations = {
+  UPDATE_QUESTIONS: 'updateQuestions',
+  SET_EVENT: 'setEvent',
+  SET_EVENT_ID: 'setEventId',
+  SET_LOADING: 'setLoading'
```

Comment:
setLoading should set the loading state on this event namespace instead of the root store.